### PR TITLE
You know it! 😊

### PR DESCRIPTION
**Pull request by alexpearce**
This commit does something very important.

It solves issue #217.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ layout: lesson
 title: First Steps in LHCb
 ---
 
+Alex is great.
```
Comment (alexpearce) [inline index.md:6]: Are you sure?

**Review by alexpearce — COMMENTED** [review]
Generally OK but not sure about this "Alex" guy.

File: index.md (hunk 1 above)
Comment (ibab) [reply index.md:6]: I'm not sure sure about this

**Comment by ibab** [comment]
I have some serious doubts about this PR, so I'll close it :P

**Pull request closed without merging**